### ISSUE TITLE
adv class is so general

### ISSUE THREAD

**Issue by zoli** [farrokhi/adblock-iran#65]
adv class is general and may get used by other things that are not advertisement

**Comment by MasterKia** [comment]
> adv class is general and may get used by other things that are not advertisement

Yes, that is also an problem with the [uBOPa filterlist](https://github.com/nimasaj/uBOPa) and I've [reported](https://github.com/nimasaj/uBOPa/issues) multiple breakage in that repo.

In uBO, you can use "unbreak syntax" to undo some filter.

**Comment by farrokhi** [comment]
Would you share some examples of broken sites?

**Comment by zoli** [comment]
> Would you share some examples of broken sites?

I opened this issue because I had problem with a site but I don't remember and couldn't find it now.

**Comment by zoli** [comment]
> Would you share some examples of broken sites?

Found it now :). https://ketabchi.com/ has an advantages section which each advantage has `adv` class.

**Comment by MasterKia** [comment]
درود @zoli 
من همه فیلتر های لیست Adblock-Iran رو بازنویسی کردم و لیست [PersianBlocker](https://github.com/MasterKia/PersianBlocker) رو به وجود آوردم.
امروز لیست PersianBlocker به لیست های پیشفرض uBlock Origin [افزوده](https://github.com/gorhill/uBlock/commit/33b839fdd03f74689df3ee2b5c25a06435b350e0) شد.
